src/components/CallView/CallView.vue

```diff
@@ -30,26 +30,23 @@
 			:local-shared-data="localSharedData" />
 
 		<template v-else>
-			<EmptyCallView v-if="!remoteParticipantsCount && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
+			<EmptyCallView v-if="!callParticipantModels.length && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
 
 			<div id="videos">
 				<template v-if="!isGrid">
 					<!-- Selected video override mode -->
 					<div v-if="showSelectedVideo"
 						class="video__promoted selected-video"
 						:class="{'full-page': isOneToOne}">
-						<template v-for="callParticipantModel in reversedCallParticipantModels">
-							<VideoVue v-if="callParticipantModel.attributes.peerId === selectedVideoPeerId"
-								:key="callParticipantModel.attributes.selectedVideoPeerId"
-								:token="token"
-								:model="callParticipantModel"
-								:shared-data="sharedDatas[selectedVideoPeerId]"
-								:show-talking-highlight="false"
-								:is-grid="true"
-								:is-big="true"
-								:is-one-to-one="isOneToOne"
-								:fit-video="true" />
-						</template>
+						<VideoVue v-if="selectedCallParticipantModel"
+							:token="token"
+							:model="selectedCallParticipantModel"
+							:shared-data="sharedDatas[selectedVideoPeerId]"
+							:show-talking-highlight="false"
+							:is-grid="true"
+							:is-big="true"
+							:is-one-to-one="isOneToOne"
+							:fit-video="true" />
```

Comment:
```suggestion
							:show-talking-highlight="false"
							is-grid
							is-big
							:is-one-to-one="isOneToOne"
							fit-video />
```

You usually never miss these 🧐